site/cypress/e2e/Cypress-Feature/polls.spec.js

```diff
@@ -364,7 +400,11 @@ describe('Test cases revolving around polls functionality', () => {
         cy.logout();
         cy.login();
         cy.visit(['sample', 'polls']);
-        cy.contains('Poll Cypress Test').siblings(':nth-child(6)').children().click();
+        //Wait 6 seconds to wait out the time remaining for poll to close
+        cy.wait(6000);
```

Comment:
I'm still a little bit concerned about whether this will lead to flaky tests down the road, but perhaps it's good enough for now.